src/ids.ts

```diff
@@ -10,7 +10,7 @@ export const generateShortId = customAlphabet(alphabet, 9);
 export const generateVerifyCode = customAlphabet('1234567890', 6);
 export const generateUUID = () => randomUUID();
 
-export const generateTrackingId = (
+export const generateTrackingId = async (
```

Comment:
customAlphabet is no longer async, but preserved with our `generateTrackingId` implementation